crates/core_arch/src/arm/neon/generated.rs

```diff
@@ -0,0 +1,4658 @@
+
+//
```

Comment:
The content of this file should also be inlined into the neon module file.

Reply:
This will make it harder to add additional intrinsics, since then the bookkeeping has to happen what intrinsics already exist and what not.

Reply:
Sure, but we can't merge a file that says "DO NOT MODIFY MANUALLY" when there is no other way to modify it.

Reply:
removed the comment, it's not like anyone will edit those anyway.